fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -474,11 +474,17 @@
     public static int stream_load_default_timeout_second = 600; // 600s
 
     /*
-     * Max load timeout applicable to all type of load
+     * Max load timeout applicable to all type of load except for stream load
      */
     @ConfField(mutable = true, masterOnly = true)
     public static int max_load_timeout_second = 259200; // 3days
 
+    /*
+     * Max stream load and streaming mini load timeout
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static int max_stream_load_timeout_second = 600; // 600s
```

Comment:
You need to set the default value to a large value, I suggest to be same as `max_load_timeout_second`.
Otherwise, after online, some existing load job may fail to be submitted.

Reply:
ok